cubedash/summary/_stores.py

```diff
@@ -547,19 +547,19 @@ def _find_product_fixed_metadata(
             if field.type_name in simple_field_types and name in first_dataset_fields
         ]
 
-        # I don't think this check is necessary?
+        # TODO: remove if confirmed unnecessary, or make less stringent
```

Comment:
I think it's better to delete the code+comments instead so we don't have to look at some commented out code for months/years.

If it turns out this PR is wrong, `git revert` will  be equally happy to restore deleted lines as it is to uncomment lines.